### PR TITLE
`Programming exercise`: Fill title and short name during import

### PR DESCRIPTION
**Pull request by milljoniaer**
#### General

- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://docs.artemis.cit.tum.de/dev/development-process/#naming-conventions-for-github-pull-requests).

#### Client
- [x] **Important**: I implemented the changes with a very good performance, prevented too many (unnecessary) REST calls and made sure the UI is responsive, even with large data.
- [x] I followed the [coding and design guidelines](https://docs.artemis.cit.tum.de/dev/guidelines/client/).
- [x] Following the [theming guidelines](https://docs.artemis.cit.tum.de/dev/guidelines/client-design/), I specified colors only in the theming variable files and checked that the changes look consistent in both the light and the dark theme.
- [x] I documented the TypeScript code using JSDoc style.
- [x] I added multiple screenshots/screencasts of my UI changes.

### Motivation and Context
Stephan requested that we should also import title and shortname for programming exercises.

### Description
This PR removes the cleanup of title and shortname during import of programming exercises.

### Steps for Testing

Prerequisites:
- 1 Instructor

1. Log in to Artemis
2. Navigate to Exercise Administration
3. Click "Import Programming Exercise"
4. Chose an existing Programming Exercise
5. Verify that title and short-name are imported as well (screenshot)

#### Code Review
- [x] Code Review 1
- [x] Code Review 2
#### Manual Tests
- [x] Test 1
- [x] Test 2

### Screenshots
image

### CODE REVIEW

**Review by JanaNF — APPROVED** [review]
Manual tested on legacy ts2. Worked as expected.

Feature idea:
Maybe you could implement this for the other exercise types too. For quizzes, this works too, but when I tried this with modeling exercises, the title was not included. Just think about it.

**Review by vinceclifford — APPROVED** [review]
Works as expected, tested on TS2.

**Review by florian-glombik — APPROVED** [review]
Code changes look good

**Review by reschandreas — DISMISSED** [review]
code lgtm

File: src/test/javascript/spec/component/programming-exercise/programming-exercise-update.component.spec.ts [hunk 1]
```diff
@@ -417,9 +417,9 @@ describe('ProgrammingExercise Management Update Component', () => {
 
             verifyImport();
 
-            // name and short name should not be imported
-            expect(comp.programmingExercise.title).toBeUndefined();
-            expect(comp.programmingExercise.shortName).toBeUndefined();
+            // name and short name should also be imported
+            expect(comp.programmingExercise.title).toEqual(programmingExercise.title);
+            expect(comp.programmingExercise.shortName).toEqual(programmingExercise.shortName);
```
Comment (JohannesStoehr) [inline src/test/javascript/spec/component/programming-exercise/programming-exercise-update.component.spec.ts:422]: Can you please move this into verifyImport() as well?

**Review by JohannesStoehr — CHANGES_REQUESTED** [review]

**Review by sarpsahinalp — APPROVED** [review]
Tested on ts2. Works as expected.

**Review by basak-akan — APPROVED** [review]
Tested on TS1, title and short-name are imported while importing programming exercise

**Review by MichaelOwenDyer — APPROVED** [review]
Code looks good to me 👍🏽

**Review by JohannesStoehr — APPROVED** [review]
Code

**Review by reschandreas — APPROVED** [review]
reapprove after code-changes

**Pull request merged**